### ISSUE TITLE
Linux support?

### ISSUE THREAD

**Issue by datlaunchystark** [exeldro/obs-audio-monitor#66]
I don't mean to put any pressure or anything however I'm curious if there's any plans to release this plugin for Linux?

**Comment by exeldro** [comment]
I tried to get it working with pulse, but without success. I need help from a developer with more Linux experience to get audio monitor working on Linux.

**Comment by datlaunchystark** [comment]
Fair enough, I did see a fork of this plugin on the Arch AUR and I saw that people were having issues with it being stable on there and it compiling.  Pulse audio is strange sometimes..

Also if you're curious, here's the fork I found on the AUR: https://aur.archlinux.org/packages/obs-audio-monitor
